src/regex.nim

```diff
@@ -2852,12 +2853,15 @@ when defined(forceRegexAtRuntime):
   template re*(s: string): Regex =
     toPattern(s)
 else:
-  template re*(s: string): Regex =
-    ## Parse and compile a regular
-    ## expression at compile-time
+  template re*(s: static string): Regex =
```

Comment:
can we make it a proc? are there pros/cons?

Reply:
done; (and nicer tracebacks still hold even with inline)